### PR TITLE
QwMockData Root Guide

### PR DESCRIPTION
**Pull request by Mrc1104**
Adds the QwMockData ROOT Guide as a markdown file.

Please send any feedback to Ryan Conaway (mrc@jlab.org).

### CODE REVIEW

File: Guide/Images/BCM_example.png
Comment (wdconinc) [inline Guide/Images/BCM_example.png:1]: Text blocks should not be uploaded as pngs. These are not searchable and hard to update by others.

Comment (Mrc1104) [reply Guide/Images/BCM_example.png:1]: I removed the majority of the images and replaced them with inline text. Please take a look and let me know if there is anything else you would like to see changed or added.

**Review by wdconinc — APPROVED** [review]

**Review by wdconinc — APPROVED** [review]

File: Guide/Guide.md [hunk 1]
```diff
@@ -0,0 +1,802 @@
...
+===========================
+
+Using the JAPAN-MOLLER repository, run the following commands in a
+terminal:
+
+        $  build/qwmockdatagenerator -r 4 -e 1:20000 --config qwparity_simple.conf \
+           --detectors mock_newdets.map --data <Data>
+        $  build/qwparity -r 4 --config qwparity_simple.conf \
+           --detectors mock_newdets.map --data-handler mock_datahandlers.map \
+           --data <Data> --rootfiles <Rootfiles>
```
Comment (paulmking) [inline Guide/Guide.md:20]: Please correct the "--data-handler" flag to be "--datahandlers".

Comment (Mrc1104) [reply Guide/Guide.md]: I corrected the typo in the --datahandlers argument.

**Comment by Mrc1104** [comment]
In commit a8414df, I corrected the qwparity --datahandlers argument typo. I also added a section on how to do basic plotting of the root trees and their branches. This includes adding a friend between two trees in two different root files. 

Please review and let me know of any feedback.

**Pull request merged**